bip-musig2.mediawiki

```diff
@@ -366,7 +367,7 @@ Algorithm ''NonceGen(sk, aggpk, m, extra_in)'':
 ** Let ''m_prefixed = bytes(1, 1) || bytes(8, len(m)) || m''
 * If the optional argument ''extra_in'' is not present:
 ** Let ''extra_in = empty_bytestring''
-* Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(aggpk)) || aggpk || m_prefixed || bytes(4, len(extra_in)) || extra_in || bytes(1, i - 1))) mod n'' for ''i = 1,2''
+* Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(pk)) || bytes(1, len(aggpk)) || aggpk || m_prefixed || bytes(4, len(extra_in)) || extra_in || bytes(1, i - 1))) mod n'' for ''i = 1,2''
```

Comment:
Is it correct that only `bytes(1, len(pk))` and not the pk itself is written? python reference code writes the pk as well

Reply:
Thanks, fixed.